### PR TITLE
Init infrastructure for pki-init

### PR DESCRIPTION
**Pull request by jim-wang-yutsung**
Initial infra-structure based on the story US4299: Create pki-init docker container. 
The executable now is just a place-holder and does nothing and it will be replaced it by golang binary later.

### CODE REVIEW

File: pkiinit/docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,549 @@
...
+
+  # new pki-init service section
+  pki-init:
+    image: edgexfoundry/docker-edgex-pki-init:1.0.0-dev
+    container_name: edgex-pki-init
+    hostname: edgex-pki-init
+    networks:
+      - edgex-network
+    volumes:
+      - pki-init-config:/usr/local/etc/degex/pki
```
Comment (bnevis-i) [inline pkiinit/docker-compose.yml]: Typo.

Comment (jim-wang-yutsung) [reply pkiinit/docker-compose.yml]: good catch

**Review by bnevis-i — APPROVED** [review]
LGTM

**Review by adam-intel — APPROVED** [review]

**Review by adam-intel — APPROVED** [review]

**Comment by jim-wang-yutsung** [comment]
=== DO NOT MERGE THIS === 
This is for review purpose only. 
There is a separate feature branch created for the whole pki-init EPIC: ```pki-initialization```

**Pull request closed without merging**